### PR TITLE
Added 12.1

### PR DESCRIPTION
**Pull request by jacobwagner**
- added all environment files
- added all interface/user_config files
- added 12.1 playbook
- need to update README.md

### CODE REVIEW

File: playbooks/rpc-12.1-playbook.yml [hunk 1]
```diff
@@ -0,0 +1,649 @@
+---
+# This playbook installs Rackspace Private Cloud v12.0
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml]: `v12.1`

File: playbooks/rpc-12.1-playbook.yml [hunk 2]
```diff
@@ -0,0 +1,649 @@
...
+          pvs: /dev/xvdz1
+
+    - name: Create /etc/network/interfaces.d/
+      file: dest=/etc/network/interfaces.d/ state=directory
+
+    - name: Source /etc/network/interfaces.d/*.cfg
+      lineinfile: dest=/etc/network/interfaces line='source /etc/network/interfaces.d/*.cfg' insertafter=EOF state=present
+
+    - name: Create /etc/network/interfaces.d/rpc.cfg
+      template: src=templates/etc/network/interfaces.d/rpc-12.0.cfg.j2 dest=/etc/network/interfaces.d/rpc.cfg
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml]: `rpc-12.1.cfg.j2`

File: playbooks/rpc-12.1-playbook.yml [hunk 3]
```diff
@@ -0,0 +1,649 @@
...
+      when: ansible_local.default_configs is not defined
+      with_items:
+        - openstack-ansible
+        - rpcd
+
+    - name: Copy Default Configuration Files Done Fact
+      ini_file: dest=/etc/ansible/facts.d/default_configs.fact section=state option=done value=true
+
+    - name: Copy OpenStack User Config File
+      template: src=templates/etc/openstack_deploy/openstack_user_config-12.0.yml.j2 dest=/etc/openstack_deploy/openstack_user_config.yml
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml]: `openstack_user_config-12.1.yml.j2`

File: playbooks/rpc-12.1-playbook.yml [hunk 4]
```diff
@@ -0,0 +1,649 @@
...
+      with_items:
+        - { regexp: '^glance_default_store:', line: 'glance_default_store: swift' }
+
+- name: Configure Stand Alone Object
+  hosts: infra[0]
+  tags:
+    - object_stand_alone
+  tasks:
+    - name: Copy OpenStack User Config File
+      template: src=templates/etc/openstack_deploy/openstack_user_config-12.0-object-stand-alone.yml.j2 dest=/etc/openstack_deploy/openstack_user_config.yml
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml]: `openstack_user_config-12.1-object-stand-alone.yml.j2`

File: playbooks/rpc-12.1-playbook.yml [hunk 5]
```diff
@@ -0,0 +1,649 @@
...
+
+- name: Configure Object Common
+  hosts: infra[0]
+  tags:
+    - object
+    - object_stand_alone
+  tasks:
+    - name: Copy OpenStack Swift Config File
+      template: src=templates/etc/openstack_deploy/conf.d/swift-12.0.yml.j2 dest=/etc/openstack_deploy/conf.d/swift.yml
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml]: `swift-12.1.yml.j2`

File: playbooks/rpc-12.1-playbook.yml [hunk 6]
```diff
@@ -0,0 +1,649 @@
...
+    - name: Set Deploy Environment Swift Fact
+      ini_file: dest=/etc/ansible/facts.d/deploy_environment.fact section=option option=deploy_swift value=yes
+
+- name: Configure Ceph
+  hosts: infra[0]
+  tags:
+    - ceph
+  tasks:
+    - name: Copy Ceph OSA Config File
+      template: src=templates/etc/openstack_deploy/conf.d/ceph-12.0.yml.j2 dest=/etc/openstack_deploy/conf.d/ceph.yml
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml]: `ceph-12.1.yml.j2`

**Comment by byronmccollum** [comment]
Hey Jacob...

Can you make sure that the 12.1 playbook references the corresponding 12.1 jinja templates...and that all the templates exist accordingly?

Cheers
Byron

**Comment by jacobwagner** [comment]
Yeah i will update this today

File: playbooks/rpc-12.1-playbook.yml [hunk 7]
```diff
@@ -0,0 +1,649 @@
...
+      template: src=templates/etc/openstack_deploy/openstack_user_config-12.1-object-stand-alone.yml.j2 dest=/etc/openstack_deploy/openstack_user_config.yml
+
+- name: Configure Object Common
+  hosts: infra[0]
+  tags:
+    - object
+    - object_stand_alone
+  tasks:
+    - name: Copy OpenStack Swift Config File
+      template: src=templates/etc/openstack_deploy/conf.d/swift-12.1.yml.j2 dest=/etc/openstack_deploy/conf.d/swift.yml
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml:465]: Can you add the `swift-12.1.yml.j2` template?

File: playbooks/rpc-12.1-playbook.yml [hunk 8]
```diff
@@ -0,0 +1,649 @@
...
+    - name: Set Deploy Environment Swift Fact
+      ini_file: dest=/etc/ansible/facts.d/deploy_environment.fact section=option option=deploy_swift value=yes
+
+- name: Configure Ceph
+  hosts: infra[0]
+  tags:
+    - ceph
+  tasks:
+    - name: Copy Ceph OSA Config File
+      template: src=templates/etc/openstack_deploy/conf.d/ceph-12.1.yml.j2 dest=/etc/openstack_deploy/conf.d/ceph.yml
```
Comment (byronmccollum) [inline playbooks/rpc-12.1-playbook.yml:479]: Can you add the `ceph-12.1.yml.j2` template?

**Comment by byronmccollum** [comment]
Thanks for your patience...

**Pull request merged**